packages/aeproject-cli/aeproject-init/init.js

```diff
@@ -90,33 +99,72 @@ const installYarn = async () => {
     createMissingFolder(constants.contractsDir);
-    copyFileOrDir(fileSource, shape ? constants.shapeContractFileDestination : constants.contractFileDestination)
+
+    const destination = shape ? constants.shapeContractFileDestination : constants.contractFileDestination;
+
+    try {
+        copyFileOrDir(fileSource, destination)
+    } catch (error) {
+        if (error.message.includes('already exists')) {
+            await prompt(error, copyFileOrDir, fileSource, destination)
+        } else {
+            throw Error(error);
+        }
+    }
 }
 
-const setupIntegrations = () => {
+const setupIntegrations = async () => {
     print(`===== Creating integrations directory =====`);
     const fileSource = `${ __dirname }${ constants.artifactsDir }/${ constants.contratsAeppSetting }`;
     createMissingFolder(constants.integrationsDir);
-    copyFileOrDir(fileSource, constants.contratsAeppSettingFileDestination)
+    
+    try {
+        copyFileOrDir(fileSource, constants.contratsAeppSettingFileDestination)
+    } catch (error) {
+        if (error.message.includes('already exists')) {
+            await prompt(error, copyFileOrDir, fileSource, constants.contratsAeppSettingFileDestination)
+        } else {
+            throw Error(error);
+        }
+    }
 }
 
-const setupTests = (shape) => {
+const setupTests = async (shape) => {
     print(`===== Creating tests directory =====`);
     const fileSource = shape ? `${ __dirname }${ constants.shapeArtifactsDir }/${ constants.shapeTestTemplateFile }` : `${ __dirname }${ constants.artifactsDir }/${ constants.testTemplateFile }`;
     createMissingFolder(constants.testDir, "Creating tests file structure");
-    copyFileOrDir(fileSource, shape ? constants.shapeTestFileDestination : constants.testFileDestination)
+    
+    try {
+        copyFileOrDir(fileSource, shape ? constants.shapeTestFileDestination : constants.testFileDestination)
+    } catch (error) {
+        if (error.message.includes('already exists')) {
+            await prompt(error, copyFileOrDir, fileSource, constants.testFileDestination)
+        } else {
+            throw Error(error);
+        }
+    }
 }
 
 const setupDeploy = async (shape) => {
 
     print(`===== Creating deploy directory =====`);
     const fileSource = shape ? `${ __dirname }${ constants.shapeArtifactsDir }/${ constants.shapeDeployTemplateFile }` : `${ __dirname }${ constants.artifactsDir }/${ constants.deployTemplateFile }`;
     createMissingFolder(constants.deployDir, "Creating deploy directory file structure");
-    copyFileOrDir(fileSource, constants.deployFileDestination)
+    
+    try {
+        copyFileOrDir(fileSource, constants.deployFileDestination)
+    } catch (error) {
+        if (error.message.includes('already exists')) {
+            await prompt(error, copyFileOrDir, fileSource, constants.deployFileDestination)
+        } else {
+            throw Error(error);
+        }
+    }
 }
 
 const setupDocker = () => {
```

Comment:
what will happen if the process has stopped on creating the docker images?

Reply:
by default we have a flag that always will overwrite existents files
```
const copyOptions = {
        overwrite: true
    }
```